app_flutter/lib/widgets/lattice.dart

```diff
@@ -545,8 +545,7 @@ class _RenderLatticeBody extends RenderBox {
     newChildParentData.coordinate = newCoordinate;
   }
 
-  void removeChild(RenderBox child) {
-    final _Coordinate coordinate = _childrenByRenderBox[child];
```

Comment:
You probably don't need `_childrenByRenderBox` anymore.

Reply:
Thanks for pointing this out! Removed.